raiden/tests/conftest.py

```diff
@@ -1,18 +1,18 @@
 # pylint: disable=wrong-import-position,redefined-outer-name,unused-wildcard-import,wildcard-import
 import re
+import sys
 
 import gevent
 import py
-import sys
-from gevent import monkey
-monkey.patch_all()
-
 import pytest
+from gevent import monkey
 
-from raiden.utils.cli import LogLevelConfigType
 from raiden.exceptions import RaidenShuttingDown
-from raiden.tests.fixtures.variables import *  # noqa: F401,F403
 from raiden.log_config import configure_logging
+from raiden.tests.fixtures.variables import *  # noqa: F401,F403
+from raiden.utils.cli import LogLevelConfigType
+
+monkey.patch_all()
```

Comment:
Same as above. This line must stay at the top of the file. (And apparently has already been in the wrong spot for a while)